opendut-edgar/src/setup/tasks/write_configuration.rs

```diff
@@ -158,6 +184,12 @@ mod tests {
             config_override: ConfigOverride {
                 peer_id: new_peer_id,
                 carl_url: Url::parse("https://example.com:1234").unwrap(),
+                auth_config: AuthConfig {
+                    issuer_url: Url::parse("https://test.com:1234").unwrap(),
+                    client_secret: ClientSecret(CLIENT_SECRET.to_string()),
+                    client_id: ClientId(CLIENT_ID.to_string()),
```

Comment:
Add serialization for auth config when OIDC enabled/disabled.

Reply:
Also changed the implementation to what you suggestion.